src/components/Dashboard/DashboardHeader.js

```diff
@@ -30,9 +30,9 @@ export default function DashboardHeader({ location, currentUser, ...props }) {
               : 'Header__tab'
           }
         >
-          <Link to="/dashboard/crossings/map">Crossings Map</Link>
+          <Link to="/dashboard/map">Crossings Map</Link>
         </li>
-        <CommunityListDropdownButton />
+        <CommunityListDropdownButton location={location}/>
```

Comment:
maybe instead of passing the entire location object in here we could do the "are we on the dashboard?" logic up here and pass down a bool for dash or not